### DISCUSSION TITLE
trouble setting up cronjob

### DISCUSSION THREAD

**Discussion by ksf0** [coccoinomane/crabada.py#33] [General]
edit: changed a lot of things and got it working, not sure what the issue was

**Comment by coccoinomane** [comment]
Glad you fixed your issue, sometimes a little bit of magic is everything that is needed 😜

**Comment by Mrooie** [comment]
If cron doesn't work on MacOs Catalina and higher, it might be a privacy setting.
Giving a full disc access to cron worked for me.
https://dccxi.com/posts/crontab-not-working-catalina/

**Comment by coccoinomane** [comment]
Thanks @Mrooie for sharing your solution 💪
